doc/source/user_guide/style.ipynb

```diff
@@ -1050,102 +1394,151 @@
+    "\n",
+    "We can see example of the HTML by calling the [.render()][render] method.\n",
+    "\n",
+    "[render]: ../reference/api/pandas.io.formats.style.Styler.render.rst"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "### Limitations\n",
-    "\n",
-    "- DataFrame only `(use Series.to_frame().style)`\n",
-    "- The index and columns must be unique\n",
-    "- No large repr, and performance isn't great; this is intended for summary DataFrames\n",
-    "- You can only style the *values*, not the index or columns (except with `table_styles` above)\n",
-    "- You can only apply styles, you can't insert new HTML entities\n",
-    "\n",
-    "Some of these will be addressed in the future.\n",
-    "Performance can suffer when adding styles to each cell in a large DataFrame.\n",
-    "It is recommended to apply table or column based styles where possible to limit overall HTML length, as well as setting a shorter UUID to avoid unnecessary repeated data transmission. \n"
+    "print(pd.DataFrame([[1,2],[3,4]], index=['i1', 'i2'], columns=['c1', 'c2']).style.render())"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Terms\n",
+    "### CSS Hierarchies\n",
     "\n",
-    "- Style function: a function that's passed into `Styler.apply` or `Styler.applymap` and returns values like `'css attribute: value'`\n",
-    "- Builtin style functions: style functions that are methods on `Styler`\n",
-    "- table style: a dictionary with the two keys `selector` and `props`. `selector` is the CSS selector that `props` will apply to. `props` is a list of `(attribute, value)` tuples. A list of table styles passed into `Styler`."
+    "The examples have shown that when CSS styles overlap, often the latter (in the HTML sense) takes precedence. So the following yield different results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df4 = pd.DataFrame([['text']])\n",
+    "df4.style.applymap(lambda x: 'color:green;')\\\n",
+    "         .applymap(lambda x: 'color:red;')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df4.style.applymap(lambda x: 'color:red;')\\\n",
+    "         .applymap(lambda x: 'color:green;')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Fun stuff\n",
+    "This is only true for CSS rules that are equivalent in hierarchy, or importance. You can read more about [CSS specificity here](https://www.w3schools.com/css/css_specificity.asp) but for our purposes it pays to summarise the key points:\n",
```

Comment:
Instead of "it pays", maybe "it suffices"?

Reply:
summarize instead of summarise

Reply:
changed. Although "summarise" is a valid spelling in my part of the world.